src/omop_etl_wrapper/database/database.py

```diff
@@ -146,7 +146,13 @@ def get_record_targets(record_containing_object: Iterable) -> Iterable[str]:
             target table
         """
         for record in record_containing_object:
-            placeholder_schema = record.__table_args__.get('schema')
+            # if table constraints present (SQLAlchemy CheckConstraint),
+            # __table_args__ consists of a tuple where the dictionary
+            # containing the schema name is listed as the last element
+            if len(record.__table_args__) > 1:
+                placeholder_schema = record.__table_args__[-1].get('schema')
+            else:
+                placeholder_schema = record.__table_args__.get('schema')
```

Comment:
this is a very funny thing I discovered, apparently a consequence of the new constraint changes

Reply:
Glad you found this! I think it can be done in a slightly simpler way by not using the `__table_args__` at all, but instead use the `__table__` variable:
```suggestion
            placeholder_schema = record.__table__.schema
```